README.md

```diff
@@ -19,8 +19,7 @@ The [TAG Contributor Strategy charter](/CHARTER.md) outlines the scope of our gr
   - [TAG Chairs](#tag-chairs)
   - [Tech Leads](#tech-leads)
   - [TOC Liaisons](#toc-liaisons)
-  - [Original Bootstrap TAG members](#original-bootstrap-tag-members)
-  - [Emeritus Leads](#emeritus-leads)
```

Comment:
These changes LGTM, but while at it:
- Can you add an entry for "Valued TAG Members"
- Can you remove "Community Infrastructure Team" entry?
- Can you adda n entry for "Contributor Strategy Overview Diagram"?

Reply:
Thanks @aliok for spotting that. Fine combed the file and found a few more nits. Should be OK for review now.